code/datums/jobs/squads.dm

```diff
@@ -384,30 +384,28 @@
 	var/text = copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN)
 	if(ishuman(sender))
 		var/obj/item/card/id/ID = sender.get_idcard()
-		nametext = "[ID?.rank] [sender.name] transmits: "
+		nametext = "[ID?.rank] [sender.real_name] transmits: "
 		text = "<font size='3'><b>[text]<b></font>"
 	return "[nametext][text]"
 
 
 /datum/squad/proc/message_squad(message, mob/living/carbon/human/sender)
-	var/text = span_notice("<B>\[Overwatch\]:</b> [format_message(message, sender)]")
-	for(var/i in marines_list)
-		var/mob/living/L = i
-		message_member(L, text, sender)
-
+	if( is_ic_filtered(message) || NON_ASCII_CHECK(message))
+		to_chat(sender, span_boldnotice("Message invalid. Check your message does not contain filtered words or characters."))
+		return
 
-/datum/squad/proc/message_leader(message, mob/living/carbon/human/sender)
-	if(!squad_leader || squad_leader.stat != CONSCIOUS || !squad_leader.client)
-		return FALSE
-	return message_member(squad_leader, span_notice("<B>\[SL Overwatch\]:</b> [format_message(message, sender)]"), sender)
+	var/header = "AUTOMATED CIC NOTICE:"
+	if(sender)
+		header = "CIC SQUAD MESSAGE FROM [sender.real_name]:"
 
+	for(var/i in marines_list)
+		var/mob/living/marine = i
+		marine.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>[header]</u></span><br>" + message, /atom/movable/screen/text/screen_text/command_order)
```

Comment:
Is there not an appropriate span define for this?

Reply:
I dont know. I can look.

Reply:
iirc no I hand rolled this for play screen text